### DISCUSSION TITLE
Blazor com .net8 na prática: Pegando o usuário da sessão

### DISCUSSION THREAD

**Discussion by EdsonSilvaGit** [balta-io/forum#480] [Tira Dúvidas] [answered]
Olá, eu gostaria de saber se para pegar o usuário Identity da seção eu utilizo o AuthenticationStateProvider ou o UserManager< ApplicationUser > e quais as vantagens em se utilizar um ou outro. Muito obrigado.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @EdsonRobertoSilva, como vai?

O UserManager é utilizado para gerenciar o usuário, editar informações e afins, o AuthenticationStateProvider é o responsável por obter o usuário da sessão.
